### DISCUSSION TITLE
Configuration for Webhook form action

### DISCUSSION THREAD

**Discussion by bugnumber9** [elementor/elementor#21768] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Webhook action doesn't have any configuration, so it's impossible to send form submissions to a destination webhook that requires a certain format of incoming data.

### Describe the solution you'd like

It would be great to have a configuration section for a webhook action. For example, something like Uncanny Automator has: 
![ua](https://user-images.githubusercontent.com/1769548/227767594-209ea604-9332-4b1c-a4c3-51688f035ecb.png)

**Comment by ajblackfire** [comment]
This is a huge issue. The webhook feature is pretty much unusable with anything other than Zapier because of this. 

The simple solution to this is to just add an option for "use form field IDs as webhook parameters" and when selected use the form field IDs instead of the labels.

Another solution would be to add a "webhook parameter" field under the advanced tab for each form field that can be filled out.

This is such a glaring issue with this feature, please fix!

**Comment by skompini** [comment]
why hasn't this issue been addressed? it's 2025 and it's still a useless option. I even asked AI how to fix it, and it tells me there are supposed to be options like Custom Headers, Remote ID, or Additional Settings. I see none of these options in Webhook, even after toggling Advanced Data to yes. What gives? Am I missing something? I've got a client who needs to send sensitive data through a form, and we've got info that has to be included. Can't do it this way. Very very annoying and makes no sense.
